modules/db/src/test/java/org/mule/module/db/integration/config/GenericDbConfigTestCase.java

```diff
@@ -27,7 +27,7 @@
 {
 
     private static final String EXPECTED_DRIVER_CLASSNAME = "org.apache.derby.jdbc.EmbeddedDriver";
-    private static final String EXPECTED_URL = "jdbc:derby:muleEmbeddedDB;sql.enforce_strict_size=true";
```

Comment:
Also, I would change the database to `target/muleEmbeddedDB` so derby files are automatically cleaned when executing a `clean` with maven.